src/tokens/TokenDetails.tsx

```diff
@@ -56,12 +59,42 @@ const MAX_ACTION_BUTTONS = 3
 export default function TokenDetailsScreen({ route }: Props) {
   const { tokenId } = route.params
   const { t } = useTranslation()
+  const dispatch = useDispatch()
   const token = useTokenInfo(tokenId)
   if (!token) throw new Error(`token with id ${tokenId} not found`)
   const actions = useActions(token)
   const tokenDetailsMoreActionsBottomSheetRef = useRef<BottomSheetRefType>(null)
   const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
 
+  // Price History Things - Maybe this should all move to the chart component and we just pass in the token id
```

Comment:
Was helpful for testing, but will be removed prior to taking this PR out of draft including the useEffect hook.